src/sentry/static/sentry/app/views/performance/constants.tsx

```diff
@@ -13,7 +12,7 @@ export const PERFORMANCE_TERMS: Record<string, string> = {
   p95: t('p95 indicates the duration that 95% of transactions are faster than.'),
   p99: t('p99 indicates the duration that 99% of transactions are faster than.'),
   userMisery: t(
-    'User misery is the percentage of users who are experiencing load times 4x the base apdex rate of 300ms.'
+    "User misery is the percentage of users who are experiencing load times 4x your organization's apdex threshold."
```

Comment:
I see that this is in constants, but is it possible to include the org threshold here too?

Reply:
We could get the threshold in. It will mean wrapping tooltip messages in a function like what I did with selector options.